From 3ff33c7cebb0feb4f8c9722b8e002ff969068480 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 23 May 2024 15:01:04 +0200
Subject: [PATCH 0844/1695] plat-stm32mp2: print configuration profile

Print stm32mp configuration profile at info level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ec0ad840a7d54943efbbfe77fcd01f377022f02b plat-stm32mp2: print configuration profile]
Change-Id: Ifc4f61433d4f4257519ac057fa753070139c881b
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/382115
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/main.c b/core/arch/arm/plat-stm32mp2/main.c
index 183d663a4..98e0dce0f 100644
--- a/core/arch/arm/plat-stm32mp2/main.c
+++ b/core/arch/arm/plat-stm32mp2/main.c
@@ -32,6 +32,7 @@
 #include <stm32_util.h>
 #include <stm32mp_pm.h>
 #include <trace.h>
+#include <util.h>
 
 /* DBGMCU registers */
 #define DBGMCU_CR			U(0x4)
@@ -68,6 +69,8 @@ static TEE_Result platform_banner(void)
 	IMSG("Platform stm32mp2: flavor %s - DT %s", ID2STR(PLATFORM_FLAVOR),
 	     ID2STR(CFG_EMBED_DTB_SOURCE_FILE));
 
+	IMSG("OP-TEE ST profile: %s", TO_STR(CFG_STM32MP_PROFILE));
+
 	return TEE_SUCCESS;
 }
 
-- 
2.47.2

